ledger-service/http-json/src/main/scala/com/digitalasset/http/PackageService.scala

```diff
@@ -62,6 +62,17 @@ private class PackageService(
       val findIface = iface.Interface.findAstInterface(Function unlift lookupIf)
       diff.transform((_, iface) => iface resolveChoicesAndIgnoreUnresolvedChoices findIface)
     }
+
+    private[this] def appendAndResolveRetroactiveInterfaces(diff: PackageStore): PackageStore = {
+      def lookupIf(packageStore: PackageStore, pkId: Ref.PackageId) =
```

Comment:
So this probably returns `Option[(Interface, Interface => Map[String,Interface])]` and I guess that is used in a GetterSetterAt? (I think this refers to you telling me earlier you did not use lenses but just functions)